### PR TITLE
Add S04E05

### PR DESCRIPTION
**Pull request by stephaniehobson**

### CODE REVIEW

File: content/episodes/2019-02-04-S04E05-the-surveillance-economy.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+number: 28
+season: 4
+seasonepisode: 5
+url: /season4/episode5/
+embed: "c70f8ee2"
+mp3: "https://dts.podtrac.com/redirect.mp3/audio.simplecast.com/c70f8ee2.mp3"
+categories: "episodes"
+host: "Manoushz"
+shownotes: |
+    [Shoshana Zuboff](https://www.shoshanazuboff.com/new/about/) is the author of [The Age of Surveillance Capitalism](https://www.amazon.com/Age-Surveillance-Capitalism-Future-Frontier/dp/1610395697/ref=sr_1_1?ie=UTF8&qid=1548643852&sr=8-1&keywords=The+Age+of+Surveillance+Capitalism/).
```
Comment (craigcook) [inline content/episodes/2019-02-04-S04E05-the-surveillance-economy.md]: We should shorten the Amazon URL to `https://www.amazon.com/Age-Surveillance-Capitalism-Future-Frontier/dp/1610395697/` without the extra parameters.

File: content/episodes/2019-02-04-S04E05-the-surveillance-economy.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+
+    Read Professor S. Matthew Liao's Op-Ed [Do You Have a Moral Duty to Leave Facebook?](https://www.nytimes.com/2018/11/24/opinion/sunday/facebook-immoral.html/) in the New York Times.
+
+    Here is Nithin Coca’s story on [fully quitting Google](https://medium.com/s/story/how-i-fully-quit-google-and-you-can-too-4c2f3f85793a/).
+
+    Mozilla is on your side. Firefox has never — and will never — sell your data. And, we make things that give you more control over your life online. If you love Facebook but hate their data collection practices, reduce what they can track about you. Try Firefox’s [Facebook Container extension](https://www.mozilla.org/en-US/firefox/facebookcontainer/), which makes it harder for Facebook to track you on the web outside of Facebook.
```
Comment (craigcook) [inline content/episodes/2019-02-04-S04E05-the-surveillance-economy.md]: Should remove the `en-US` from this URL.

File: content/episodes/2019-02-04-S04E05-the-surveillance-economy.md [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+
+    **Jeanne Phillips:** To It's Complicated,
+    A close friend does not disclose the private chats that you're having. You can't trust a person like this.You can't take back what's out there. A person who mistreats you isn't a friend. Somebody who uses you is not a friend. In the future, be careful what you reveal to this so-called friend, who doesn't seem like much a friend to me at all.
+
+    **Manoush Z.:* Dear Abby’s tough love makes it obvious: on the internet, we let tech companies get away with more bad behavior than we’d ever let our real life friends get away with. The Big Tech companies especially - like Google, Facebook, Amazon and so on.And the relationships we all have with these companies have fueled an entire digital economy. An economy where     companies watch everything we say and do, and then turn that         knowledge into profit. This online data economy is so pervasive - and so lucrative - that author Shoshana Zuboff has coined a new word for it: Surveillance Capitalism.
```
Comment (craigcook) [inline content/episodes/2019-02-04-S04E05-the-surveillance-economy.md]: Missing a second `*` after Manoush's name. 
image

**Comment by stephaniehobson** [comment]
Updated.

File: content/episodes/2019-02-04-S04E05-the-surveillance-economy.md [hunk 4]
```diff
@@ -36,18 +36,17 @@ transcript: |
...
 
     In a moment, Shoshana explains what Surveillance Capitalism is and how it is shaping, and modifying our online and offline behavior. And then, we’ll explore if we should cut ties with these companies… or if we even could cut those ties if we wanted to.
 
     I’m Manoush Zomorodi. This is IRL: Online Life is Real Life. An original podcast from Mozilla.
 
     Mozilla fights for a safe and open Internet that everyone can enjoy. You can support that mission, by trying out the Firefox browser. Firefox never sells your data. Download it at firefox dot com.
 
-    **Manoush:** Shoshana Zuboff doesn’t think Google, Facebook, or the others are our friends. She believes the online world they’ve created doesn’t leave much room for pleasantries. I mean, it’s right there in the title of her book.
+    **Manoush Z.:** Shoshana Zuboff doesn’t think Google, Facebook, or the others are our friends. She believes the online world they’ve created doesn’t leave much room for pleasantries. I mean, it’s right there in the title of her book.
```
Comment (craigcook) [inline content/episodes/2019-02-04-S04E05-the-surveillance-economy.md]: Oo good catch, I missed this one ⚡️

**Pull request merged**